keyboards/bbrfkr/dynamis/dynamis.c

```diff
@@ -16,18 +16,9 @@
 
 #include "dynamis.h"
 
-#ifdef ENCODER_ENABLE
+#ifndef ENCODER_ENABLE
```

Comment:
I suspect that you meant ifdef here, (and may be my fault)

```suggestion
#ifdef ENCODER_ENABLE
```

Reply:
Ok. I fixed it.